### PR TITLE
assignment01 complete

### PR DESCRIPTION
**Pull request by christopeak**

### CODE REVIEW

**Comment by christopeak** [comment]
Just realized I failed to tag this submission with my real name: Chris Peak.

File: assignments/assignment01.rb [hunk 1]
```diff
@@ -38,18 +38,17 @@ def number_to_string(n, lang)
...
-  caps = []
-  words.each do |word|
-    caps << word.capitalize
-  end
-  caps.join " "
+  my_words = s.split(" ")
+  cap_sentence = ""
+  my_words.each {|w| cap_sentence << w.capitalize << " "} 
+  cap_sentence.chop
+
```
Comment (planetcohen) [inline assignments/assignment01.rb:45]: well done

File: assignments/assignment01.rb [hunk 2]
```diff
@@ -58,14 +57,13 @@ def titleize(s)
...
-  while n > 0
-    n -= 1
-    output << letters[n]
+  s_array = s.split("")
+  s_array_reverse = []
+  s_array.each { |c| s_array_reverse.insert(0, c)
   end
-  output
+  s_reverse = s_array_reverse.join
+
```
Comment (planetcohen) [inline assignments/assignment01.rb:65]: looks good

File: assignments/assignment01.rb [hunk 3]
```diff
@@ -80,8 +78,15 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  stripped = s.delete(" ").delete(",").downcase
-  stripped == stripped.reverse
+  s1 = s.sub(' ', '').downcase
+  s2 = s1.reverse
+
+  if s2 == s1
```
Comment (planetcohen) [inline assignments/assignment01.rb:84]: You don't need to do an if statement here. Your goal is to return true or false, depending on whether s2 == s1. The line "s2 == s1" will do exactly that.

File: assignments/assignment01.rb [hunk 4]
```diff
@@ -80,8 +78,15 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  stripped = s.delete(" ").delete(",").downcase
-  stripped == stripped.reverse
+  s1 = s.sub(' ', '').downcase
```
Comment (planetcohen) [inline assignments/assignment01.rb:81]: you also need to get rid of commas and other punctuation

File: assignments/assignment02.rb [hunk 5]
```diff
@@ -32,20 +29,16 @@ def to_sentence(ary)
...
-  if len.odd?
-    sorted_ary[mid_index]
+  alen = ary.length
+  if alen % 2 == 1 then
+  	med = ary[alen/2]
   else
-    mid_lo = sorted_ary[mid_index]
-    mid_hi = sorted_ary[mid_index+1]
-    (mid_lo + mid_hi)/2.0
+  	mean([ary[alen/2],ary[(alen/2)+1]])
```
Comment (planetcohen) [inline assignments/assignment02.rb:41]: nice reusing a method you've previously written!

**Comment by planetcohen** [comment]
Nice job!

You have not yet completed the rewrite of the Bank Statement assignment using classes instead of hashes.

**Pull request closed without merging**